README.md

```diff
@@ -36,14 +36,24 @@ network.
 
 For normal usage, you probably only need 2 or 3 peers.
 
-### TLS peers
+### Protocols
 
-As of Yggdrasil v0.3.11, peering connections over TLS are now possible. This hides
-the peering connection inside a regular TLS session, which can help in some cases
-where firewalls or deep packet inspection may identify or block regular Yggdrasil
-peering traffic.
+Currently Yggdrasil only operates as overlay network, so it requires underlaying networking.
+As for 0.4.5, following protocols supoorted for peering:
 
-TLS public peers are identified by the prefix `tls://` instead of `tcp://`. 
+- Plain TCP (`tcp://host:port`)
+- TLS (`tls://host:port`)
+- TCP over SOCKS5 (`socks://user:pass@proxy/host:port`)
```

Comment:
not TLS over SOCKS5? (i may be misremembering this one)

Reply:
can't exactly find it myself right now so probably just misremembering.

Reply:
I looked up on source code before making this PR, and there's no TLS over SOCKS5